section3/exercises/arrays.rb

```diff
@@ -77,11 +87,9 @@
 posts = ["image at beach", "holiday party", "adorable puppy", "video of cute baby"];
 
 # YOU DO: Think of a web application you commonly use. Where do you see LISTS utilized, where arrays
-# may be storing data? Come up with 3 examples - they could be from different web applications or 
+# may be storing data? Come up with 3 examples - they could be from different web applications or
 # all from the same one.
 
-# 1:
-# 2:
-# 3:
-
-
+# 1: airbnb
+# 2: vrbo
+# 3: craigslist
```

Comment:
These examples should be an example of an array data set used by the websites you've listed. For example, what array data would airbnb potentially have?